.github/workflows/java.yml

```diff
@@ -3,7 +3,7 @@ name: Scala and Java build
 on:
   push:
     branches:
-      - master
+      - '*'
```

Comment:
Do we need to change that ? :) What's a benefit from that ?

Reply:
Was just temporary so I could get the tests to run before making the PR, will change back